cli/register/register.go

```diff
@@ -201,8 +201,12 @@ func DoIt(org, pattern, nodeIdTok, userPw, email, inputFile string, nodeOrgFromF
 		if exchangePattern != "" && cliutils.AddOrg(org, pattern) != cliutils.AddOrg(org, exchangePattern) {
 			cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("Cannot proceed with the given pattern %s because it is different from the pattern %s defined for the node in the exchange.", pattern, exchangePattern))
 		} else {
-			msgPrinter.Printf("Will proceeed with the given pattern %s.", pattern)
-			msgPrinter.Println()
+			if len(inputFileStruct.Services) == 0 {
```

Comment:
Should check the service in the pattern instead of inputFile. The input file contains the user settings for the services.